desktop-src/dataxchg/html-clipboard-format.md

```diff
@@ -21,244 +21,346 @@ ms.date: 05/31/2018
+#### HTML Text
 
-            <!-- EndFragment -->
+> ```
+> <body>This is normal. <b>This is bold.</b> <i><b>This is bold italic.</b> This is italic.</i></body>
+> ```
 
-            &lt;/BODY&gt;
+This will appear as follows:
 
-            &lt;/HTML&gt;
+> This is normal. **This is bold.**  **_This is bold italic._**  _This is italic._
 
-        -   In this scenario only the BODY tag and the HTML tag appear in the context as it precedes the selected fragment. Note that start tags and end tags are included in the context. The selection, as delimited by StartSelection and EndSelection, is shown in bold.
+<!-- (TODO: Screenshot of Word showing the selected text *before* copying? omg)
+(TODO: Screenshot of Word after pasting it) --> 
 
-2.  Fragment of a table in HTML.
-    -   -   HTML text:
+When your user has loaded the above HTML text into an MSHTML-based application (MSHTML, _aka_ Trident, was Internet Explorer's engine), MSHTML handles copying a substring of HTML as follows:
 
-            &lt;BODY&gt;<TABLE BORDER><TR><TH ROWSPAN=2>Head1</TH><TD>Item 1</TD><TD>Item 2</TD><TD>Item 3</TD><TD>Item 4</TD></TR><TR><TD>Item 5</TD><TD>Item 6</TD><TD>Item 7</TD><TD>Item 8</TD></TR><TR><TH>Head2</TH><TD>Item 9</TD><TD>Item 10</TD><TD>Item 11</TD><TD>Item 12</TD></TR></TABLE>&lt;/BODY&gt;
+1. The user selects a text from the start of the word "bold" (in "This is bold.") through to the end of the word "This" (in "This is italic.") without any leading nor trailing whitespace in the selection,  
+   i.e. The selection is represented by the square-brackets in `"This is normal **This is \[bold**  **_This is bold italic_**  _This\] is italic_"`.
+2. The user clicks the Copy command button to copy the selection to the clipboard.
+3. MSHTML will consequently place this HTML text into the Windows Clipboard:
 
-        -   Appears as: ><TABLE BORDER><TR><TH ROWSPAN=2>Head1</TH><TD>Item 1</TD><TD>Item 2</TD><TD>Item 3</TD><TD>Item 4</TD></TR><TR><TD>Item 5</TD><TD>Item 6</TD><TD>Item 7</TD><TD>Item 8</TD></TR><TR><TH>Head2</TH><TD>Item 9</TD><TD>Item 10</TD><TD>Item 11</TD><TD>Item 12</TD></TR></TABLE><!\[CDATA\[\]\]>
-        -   The Item 6, Item7, Item 10, and Item 11 elements of the table are selected as a block and copied to the clipboard.
-        -   This is what will be on the clipboard (note this is IE4/MSHTML's interpretation):
+      ```
+      Version:1.0
+      StartHTML:0121
+      EndHTML:0272
+      StartFragment:0006
+      EndFragment:0106
+      StartSelection:0180
+      EndSelection:0225
+      <html><!--StartFragment--><body>This is normal. <b>This is bold.</b> <i><b>This is bold italic.</b> This is italic.</i></body><!--EndFragment--></html>
+      ```
 
-            <!DOCTYPE ...>
 
-            &lt;HTML&gt;&lt;BODY&gt;<TABLE BORDER>
+### Scenario 2 - Fragment of a table in HTML
 
-            <!--StartFragment-->
+<!-- (Please do not use these uppercase mid-1990s HTML tables, aiieee) -->
```

Comment:
```suggestion
```

after looking at https://learn.microsoft.com/en-us/windows/win32/dataxchg/html-clipboard-format, I know what you mean :)